ethcore/src/trace/db.rs

```diff
@@ -286,19 +209,8 @@ impl<T> TraceDatabase for TraceDB<T> where T: DatabaseExtras {
 				})
 				.collect();
 
-			let chain = BloomGroupChain::new(self.bloom_config, self);
-			let trace_blooms = chain.replace(&replaced_range, enacted_blooms);
-			let blooms_to_insert = trace_blooms.into_iter()
-				.map(|p| (From::from(p.0), From::from(p.1)))
-				.collect::<HashMap<TraceGroupPosition, blooms::BloomGroup>>();
-
-			let blooms_keys: Vec<_> = blooms_to_insert.keys().cloned().collect();
-			let mut blooms = self.blooms.write();
-			batch.extend_with_cache(db::COL_TRACE, &mut *blooms, blooms_to_insert, CacheUpdatePolicy::Remove);
-			// note_used must be called after locking blooms to avoid cache/traces deadlock on garbage collection
-			for key in blooms_keys {
-				self.note_used(CacheId::Bloom(key));
-			}
+			// TODO: replace it with database for trace blooms
```

Comment:
Is this supposed to be fixed in this `PR`?

Reply:
oh, yes, obsolete comment 😅